src/lib/kwlib.sh

```diff
@@ -118,6 +118,14 @@ function cmd_manager()
   eval "$command_for_eval"
 }
 
+function show_verbose()
+{
+  local flag="$1"
+  local cmd="$2"
+
+  [[ "$flag" == 'VERBOSE' ]] && printf '%s\n' "$cmd"
```

Comment:
Wouldn't it be better to use `say "$cmd"` here? It would ensure that the output adheres to the blue color standard of the verbose mode.